lib/response.ts

```diff
@@ -383,3 +383,19 @@ export class IssueProxyResponse {
     readonly redirectUri?: string
   ) { }
 }
+
+export class TxMessageWithDetailResponse {
+  constructor(
+    readonly msgIndex: number,
+    readonly requestType: string,
+    readonly details: any
+  ) { }
+}
+
+export class TxMessageListResponse {
+  constructor(
+    readonly messages: Array<TxMessageWithDetailResponse>,
+    readonly prePageToken?: string,
+    readonly nextPageToken?: string,
```

Comment:
prePageToken and nextPageToken fields seem to be returning empty strings instead of null.